apps/dashr-circos/app.R

```diff
@@ -0,0 +1,283 @@
+library(dash)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(jsonlite)
+library(dashBio)
+library(GEOquery)
+
+
+appName <- Sys.getenv("DASH_APP_NAME")
+if (appName != ""){
+  pathPrefix <- sprintf("/%s/", appName)
+  
+  Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+             DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+  
+  setwd(sprintf("/app/apps/%s", appName))
+}
+
+app <- Dash$new()
+
+
+
+source("assets/CircosFunctions.R")
```

Comment:
Running this line throws an `Error in eval(ei, envir) : object 'circos_dataframe' not found` message. Maybe the `circos_dataframe` line below has to be listed before this one?